StRoot/StPicoEvent/StPicoArrays.cxx

```diff
@@ -76,6 +80,8 @@ int StPicoArrays::picoArraySizes [NAllPicoArrays] = { 1,    // StPicoEvent
                                                       100,  // StPicoBEmcSmdPHit
                                                       100,  // StPicoETofHit
                                                       100,  // StPicoETofPidTraits
+                                                      10,  // StPicoFwdTrack
```

Comment:
I suggest to move these two lines after the MC information. It will be easier to maintain backward compatibility.

Reply:
Moved in [61525da](https://github.com/star-bnl/star-sw/pull/639/commits/61525da6d3206ad43d1a79b14d9a0813758cd469)